docarray/index/backends/in_memory.py

```diff
@@ -433,6 +433,9 @@ def _text_search_batched(
     ) -> _FindResultBatched:
         raise NotImplementedError(f'{type(self)} does not support text search.')
 
+    def __contains__(self, item: BaseDoc):
```

Comment:
```suggestion
    def _contains(self, item: BaseDoc):
```

the syntax `__smth__` means the function is a very special function, I think in this case the function should just be private and therefore be `_smth`

Reply:
this contains operator allows user to do ' doc.id in index` in a very nice way

Reply:
my bad